src/components/counter/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import { db } from '../../lib/firebase.js';
+
+function Counter({ id, name, currentValue }) {
+  const incrementValue = () => {
+    // + 1
+    currentValue = currentValue + 1;
+    // Update value in Firebase
+    var counterRef = db.collection('counters').doc(id);
+
+    counterRef
+      .update({
+        value: currentValue,
+      })
+      .then(function () {
+        console.log('Document successfully updated!');
+      })
+      .catch(function (error) {
```

Comment:
nice! love seeing a good catch for errors

Reply:
Thank you for your great comments! 😊